src/bp/dbc/pubkey.rs

```diff
@@ -116,18 +116,18 @@ where
     /// 1. Construct a byte string `lnbp1_msg`, composed of the original message
     ///    prefixed with a single SHA256 hash of `LNPBP1`
     ///    string and a single SHA256 hash of protocol-specific tag:
-    ///    `lnbp1_msg = SHA256("LNPBP1") || SHA256(<protocol-specific-tag>) ||
-    /// msg` 2. Compute HMAC-SHA256 of the `lnbp1_msg` and `P`, named
-    /// **tweaking    factor**: `f = HMAC_SHA256(s, P)`
+    ///    `lnbp1_msg = SHA256("LNPBP1") || SHA256(<protocol-specific-tag>) ||msg` 
```

Comment:
Can you pls accept the proposed change here, otherwise rust linter gives us hard life :)

```suggestion
    ///    `lnbp1_msg = SHA256("LNPBP1")||SHA256(<protocol-specific-tag>)||msg`
```